src/core/libmaven/mzMassCalculator.h

```diff
@@ -37,6 +38,9 @@ class MassCalculator {
     public:
         enum IonizationType { ESI=0, EI=1};
         static IonizationType ionizationType;
```

Comment:
El-MAVEN does not do adduct calculation yet. Not sure if we need this piece of code for now. You can leave it in for this release but make a note to remove this later?

Reply:
I will add a todo in the code itself.